### ISSUE TITLE
JPEG: change readme

### ISSUE THREAD

**Issue by bluddy** [rlepigre/ocaml-imagelib#46]
Looks like jpeg isn't supported in the code, but the readme says it is. This is very confusing to anyone looking for an image library. Am I mistaken?

**Comment by rlepigre** [comment]
Yeah, the README is not super clear, but it does say that only computing the image size is supported natively. The unix-enabled version of the library uses convert to turn JPEG images to PNG.

**Comment by bluddy** [comment]
Somebody complained that it blew up for him on discord. It's possible that he tried to open a large JPEG that can't easily be converted to PNG due to lack of appropriate compression.

**Comment by cfcs** [comment]
@bluddy I updated the README in response to your other report (#47). Do you think that it's addressed there?

RE: conversion blowing up: It would a lot easier to track down the root cause if they would post a report here, ideally having set `export OCAMLRUNPARAM=b` in their shell environment first.
The `imagelib` pixel representation is far from ideal, with the result that we sometimes end up using a lot of memory and we take a lot of performance hits from poor cache locality because pixel data is stored in separate arrays (something that could be improved upon), but it would be nice to know if it is caused by excessive memory consumption or a bug in the code. If we can close this issue and continue that conversation in another that would be easier to manage.
